apps/website/src/pages/api/show-and-tell/revalidate.ts

```diff
@@ -4,6 +4,9 @@ import { env } from "@/env";
 
 import timingSafeCompareString from "@/server/utils/timing-safe-compare-string";
 
+/**
+ * @deprecated 2025-09-15
+ */
```

Comment:
Can you file an issue to track removing this? We can pull Vercel logs to check no other traffic is going to this that we've forgotten about.